README.md

```diff
@@ -1,24 +1,24 @@
+An architecture developed and revised by the [FilledStacks](https://www.youtube.com/filledstacks) community. This architecture was initially a version of MVVM as [described in this video](https://youtu.be/kDEflMYTFlk). Since then Filledstacks app development team has built 6 production applications with various requirements. This experience along with countless requests for improvements and common functionality is what sparked the creation of this architecture package. It aims to provide **common functionalities to make app development easier** as well as code principles to use during development to ensure your code stays maintainable.
 
-**If you're Reading this disclaimer the series that does a deepdive on this architecture has not been published yet.**
+**If you're Reading this disclaimer the series that does a deep-dive on this architecture has not been published yet.**
 
 ## How Does it work
 
 The architecture is very simple. It consists of 3 major pieces, everything else is up to your implementation style. These pieces are:
 
-- **View**: Shows the UI to the user. Single widgets also qualifies as views (for consistency in terminology) a view in this case is not a "Page" it's just a UI representation.
-- **ViewModel**: Manages the state of the View, business logic and any other logic as required from user interaction. It does this by making use of the services
-- **Services**: A wrapper of a single functionality / feature set. This is commonly used to wrap things like showing a dialog, wrapping database functionality, integrating an api, etc.
+- **View**: Shows the UI to the user. Single widgets also qualify as views (for consistency in terminology) a view, in this case, is not a "Page" it's just a UI representation.
+- **ViewModel**: Manages the state of the View, business logic, and any other logic as required from user interaction. It does this by making use of the services
+- **Services**: A wrapper of a single functionality/feature set. This is commonly used to wrap things like showing a dialog, wrapping database functionality, integrating an API, etc.
 
-Lets go over some of those principles to follow during development.
+Let's go over some of those principles to follow during development.
 
-- Views should never MAKE USE of a service directly.
+- Views should never MAKE USE of service directly.
 - Views should contain zero to (preferred) no logic. If the logic is from UI only items then we do the least amount of required logic and pass the rest to the ViewModel.
 - Views should ONLY render the state in its ViewModel.
-- ViewModels for widgets that represent page views are bound to a single View only.
-- ViewModels may be re-used if the UI required the exact same functionality.
+- ViewModels for widgets that represent page views are bound to a single view only.
```

Comment:
Add back the capital for View. It's a name in this case, the name of one of the three components in the state management architecture so it starts with a capital